plik2.txt

```diff
@@ -1 +1 @@
-
+I wont to have ferrari in my garage
```

Comment:
It will was to be Lamborghini not ferrari

Reply:
ok